suites/quincy/cephfs/tier-2_cephfs_test-volume-management.yaml

```diff
@@ -355,3 +355,9 @@ tests:
       polarion-id: CEPH-83604079
       desc: cephfs subvolumegroup scenarios
       abort-on-fail: false
+  - test:
+      name: cephfs subvolume idempoence earmark
+      module: cephfs_vol_management.cephfs_vol_mgmt_subvol_idempotence_earmark.py
+      polarion-id: CEPH-83604184
+      desc: cephfs subvolume idempoence earmark
+      abort-on-fail: false
```

Comment:
is the earmark feature supported for reef and quincy release too? If yes, please ignore, else please remove automation from those releases

Reply:
it was handled in the script. If is it squid, it goes to the earmark condition else it just ends the script.